src/storybook/story-to-test/index.test.ts

```diff
@@ -30,22 +35,24 @@ describe("storybook/story-to-test", () => {
         jest.spyOn(path, "resolve").mockImplementation((_, storyPath) => storyPath);
         const storyFirst = { importPath: "./story/path/story-first.js" } as StorybookStoryExtended;
         const storySecond = { importPath: "./story/path/story-second.js" } as StorybookStoryExtended;
-
-        const storyTestFiles = await buildStoryTestFiles([storyFirst, storySecond], {
+        const opts = {
             autoScreenshots: true,
+            autoscreenshotSelector: "foobar",
             autoScreenshotStorybookGlobals: { foo: { bar: "baz" } },
-        });
+        };
+
+        const storyTestFiles = await buildStoryTestFiles([storyFirst, storySecond], opts);
 
         expect(writeStoryTestsFile).toBeCalledWith({
             testFile: "./story/path/story-first.js.testplane.js",
-            opts: { autoScreenshots: true, autoScreenshotStorybookGlobals: { foo: { bar: "baz" } } },
             stories: [storyFirst],
+            opts,
         });
 
         expect(writeStoryTestsFile).toBeCalledWith({
             testFile: "./story/path/story-second.js.testplane.js",
-            opts: { autoScreenshots: true, autoScreenshotStorybookGlobals: { foo: { bar: "baz" } } },
             stories: [storySecond],
+            opts,
         });
 
         expect(storyTestFiles).toEqual([
```

Comment:
It can not be tested with unit tests, as this code is inside story-runner, so the file just generates testplane tests (so its just a file with auto-executed function, which generates `it`s and `describe`s)